Adding autofix for core in dbt platform (#7920)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

#### Editor tab menu
  To interact with open editor tabs, right-click any tab to access the helpful options in the file tab menu.
  <Lightbox src="/img/docs/dbt-cloud/cloud-ide/editor-tab-menu-with-save.png" width="90%" title=" Right-click a tab to view the Editor tab menu options"/>

#### File Search
  You can easily search for and navigate between files using the File Navigation menu, which can be accessed by pressing Command-O or Control-O or clicking on the 🔍 icon in the File <Constant name="explorer" />.
  <Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-file-search-with-save.png" width="100%" title="The Command History returns a log and detail of all your dbt invocations."/>

#### Global Command Palette
  The Global Command Palette provides helpful shortcuts to interact with the <Constant name="cloud_ide" />, such as git actions, specialized dbt commands, and compile, and preview actions, among others. To open the menu, use Command-P or Control-P.
  <Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-global-command-palette-with-save.png" width="100%" title="The Command History returns a log and detail of all your dbt invocations."/>

> #### <Constant name="cloud_ide" /> Status modal

Reviewer: We usually don't use constants in headers, surprised this works!

Author: I think you can't easily link to it.

  <Lightbox src="/img/docs/dbt-cloud/cloud-ide/prune-branch-modal.png" width="60%" title="The Prune branches modal allows users to delete local branches that have already been deleted from the remote repository."/>

#### Revert Uncommitted Changes modal
  The Revert Uncommitted Changes modal is how users revert changes in the IDE. This is accessible via the `Revert File` option above the Version Control Options menu, or via the Git Actions button when there are saved, uncommitted changes in the IDE.
  <Lightbox src="/img/docs/dbt-cloud/cloud-ide/revert-uncommitted-changes-with-save.png" width="90%" title="The Commit Changes modal is how users change their branch."/>

#### <Constant name="cloud_ide" /> Options menu

>   The <Constant name="cloud_ide" /> Options menu can be accessed by clicking on the three-dot menu located at the bottom right corner of the <Constant name="cloud_ide" />. This menu contains global options:

Reviewer: ```suggestion
  Access the <Constant name="cloud_ide" /> Options menu by clicking the three-dot menu located at the bottom right corner of the <Constant name="cloud_ide" />. This menu contains global options:
```

>   * Viewing status details, including the <Constant name="cloud_ide" /> Status modal.

Reviewer: ```suggestion
  * View status details, including the <Constant name="cloud_ide" /> Status modal
```

>   * Restarting the <Constant name="cloud_ide" />

Reviewer: ```suggestion
  * Restart the <Constant name="cloud_ide" />
```

  * Reinstall dependencies
  * Clean dbt project
  * [Check & fix deprecations](/docs/cloud/dbt-cloud-ide/autofix-deprecations)

>   * Rollback your repo to remote, to refresh your git state and view status details

Reviewer: ```suggestion
  * Rollback your repo to remote to refresh your git state and view status details
```
